lib/findable.rb

```diff
@@ -1,22 +1,26 @@
 module Dor::Assembly
   module Findable
+    attr_reader :folder_style
```

Comment:
I'm wondering whether we should use the functionality in `DruidTools::Druid` for this searching. I think, but not sure, you can do this:

```
druid = DruidTools::Druid.new('druid:aa111bb2222', basedir)
druid.find_content(basedir) # searches multiple folder styles to locate the correct path
```

See https://github.com/sul-dlss/druid-tools/blob/master/lib/druid_tools/druid.rb#L37-L39

Same with the `path_to_xxx_file` methods below